modules/installation-configuration-parameters.adoc

```diff
@@ -1260,6 +1272,14 @@ The `licenses` parameter is a deprecated field and nested virtualization is enab
 |For control plane machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
+|`controlPlane.platform.gcp.osImage.project`
+|Optional. By default, the installation program downloads and installs the {op-system-first} image that is used to boot control plane machines. You can override the default behavior by specifying the location of a custom {op-system} image for the installation program to use for control plane machines only.
```

Comment:
I can't find specific guidance but have seen "Optional:" used for parameter descriptions, with a colon instead of a period.

Reply:
I agree that I have seen Optional used with a colon for conditional steps, but I believe the usage here is OK.